src/routes/LazyLoadRoutes.tsx

```diff
@@ -2,8 +2,8 @@ import PagePreloder from "@/components/common/PagePreloder";
 import React, { lazy, Suspense } from "react"
 
 
-export default function LazyLoadRoutes(componentName: string, url = '../pages/') {
-    const LazyElement = lazy(() => import(`${url}${componentName}.tsx`));
+export default function LazyLoadRoutes(componentName: string) {
+    const LazyElement = lazy(() => import(`../pages/${componentName}.tsx`));
 
```

Comment:
why don't we design like that 
`LazyLoadRoutes(url:string);`

and the usage will be: 

```
const productPage = "/pages/Product/index.tsx";

LazyLoadRoutes(productPage);
```